src/chainstate/stacks/miner.rs

```diff
@@ -641,13 +647,19 @@ impl StacksBlockBuilder {
 
                 considered.insert(txinfo.tx.txid());
 
-                match builder.try_mine_tx_with_len(&mut epoch_tx, &txinfo.tx, txinfo.metadata.len) {
+                let err = builder.try_mine_tx_with_len(&mut epoch_tx, &txinfo.tx, txinfo.metadata.len);
```

Comment:
Can you change this back to `match builder.try_mine_tx_with_len(&mut epoch_tx, &txinfo.tx, txinfo.metadata.len) {`, instead of `let err = ...`?

Reply:
yes, was going to clean that up.